sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

```diff
@@ -1,47 +1,90 @@
 {
```

Comment:
<p>Do you want to keep all the outputs in this notebook? And all the warnings/info outputs later on? It doesn't seem like they're adding key information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I would beg to disagree... I would have loved to see what might go wrong, as I'm getting plenty of warnings and errors myself...

Reply:
@hovanesgasparian You can still view the previous commits with the outputs. Linked below. Please open an issue if you encounter other errors and we will attempt to replicate and fix.
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/d20140ed1eb81691100607e3a8711ddd65d03681/
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/950900ff773049dc2b0caa80e1377f5a41671172/
https://app.reviewnb.com/aws/amazon-sagemaker-examples/commit/bf197a9d46d0b2799472b4cccebddded5188ce7e/

Reply:
Oh awesome, thank you! I tried that before, and got the Jedi Octocat 404 page, so I thought I just didn't have permission!